pom.xml

```
@@ -24,6 +25,12 @@
             <artifactId>mysql-connector-java</artifactId>
             <version>8.0.22</version>
         </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>3.0.1</version>
```

we shouldn't use alpha version 
```suggestion
            <version>3.0.1</version>
```